src/landscape.jl

```diff
@@ -0,0 +1,20 @@
+
+rescale!(mat) = (mat .- minimum(mat)) ./ (maximum(mat) - minimum(mat))
```

Comment:
I still think we might use the StatsBase implementation instead

Reply:
absolutely, yes

Reply:
I just asked on the StatsBase PR that added standardization and they suggest using the simple broadcasting here https://github.com/JuliaStats/StatsBase.jl/pull/85#issuecomment-777213575